src/main/app/initialize.ts

```diff
@@ -183,7 +183,7 @@ function initializeAppEventListeners() {
     app.on('before-quit', handleAppBeforeQuit);
     app.on('certificate-error', handleAppCertificateError);
     app.on('select-client-certificate', CertificateManager.handleSelectCertificate);
-    app.on('gpu-process-crashed', handleAppGPUProcessCrashed);
+    app.on('child-process-gone', handleChildProcessGone);
```

Comment:
Can we handle both events using this handler? Not sure if they differ at all but just to make sure we cover all bases.